config/config.go

```diff
@@ -80,7 +80,7 @@ type TerragruntConfig struct {
 	IsPartial bool
 
 	// Map of processed includes
-	ProcessedIncludes map[string]IncludeConfig
+	ProcessedIncludes IncludeConfigs
```

Comment:
I was wondering if should be used a pointer to `IncludeConfigs`

Reply:
If you meant:
* `ProcessedIncludes *IncludeConfigs` - _maps_ are reference types, so they are always passed by reference. We don't need a pointer. 
* `type IncludeConfigs map[string]*IncludeConfig` - yes, it will be more correct, but I suggest changing it next time